From af839d61b179836de037ec619b43bae7e62d105d Mon Sep 17 00:00:00 2001
From: Eonfge <33983090+Eonfge@users.noreply.github.com>
Date: Fri, 27 Dec 2024 09:37:20 +0100
Subject: [PATCH] Add required meta-data (#1464)

* Update io.github.freedoom.FreeDM.metainfo.xml

* Update io.github.freedoom.Phase1.metainfo.xml

* Update io.github.freedoom.Phase2.metainfo.xml
---
 dist/io.github.freedoom.FreeDM.metainfo.xml | 1 +
 dist/io.github.freedoom.Phase1.metainfo.xml | 1 +
 dist/io.github.freedoom.Phase2.metainfo.xml | 1 +
 3 files changed, 3 insertions(+)

diff --git a/dist/io.github.freedoom.FreeDM.metainfo.xml b/dist/io.github.freedoom.FreeDM.metainfo.xml
index 9856c875..98d6c825 100644
--- a/dist/io.github.freedoom.FreeDM.metainfo.xml
+++ b/dist/io.github.freedoom.FreeDM.metainfo.xml
@@ -10,6 +10,7 @@
   <developer_name>Freedoom Project</developer_name>
   <url type="homepage">https://freedoom.github.io/</url>
   <url type="bugtracker">https://github.com/freedoom/freedoom/issues</url>
+  <launchable type="desktop-id">io.github.freedoom.FreeDM.desktop</launchable>
   <description>
     <p>
       FreeDM is a fast-paced set of levels designed for multiplayer
diff --git a/dist/io.github.freedoom.Phase1.metainfo.xml b/dist/io.github.freedoom.Phase1.metainfo.xml
index a3e21d00..5b32483d 100644
--- a/dist/io.github.freedoom.Phase1.metainfo.xml
+++ b/dist/io.github.freedoom.Phase1.metainfo.xml
@@ -10,6 +10,7 @@
   <developer_name>Freedoom Project</developer_name>
   <url type="homepage">https://freedoom.github.io/</url>
   <url type="bugtracker">https://github.com/freedoom/freedoom/issues</url>
+  <launchable type="desktop-id">io.github.freedoom.Phase1.desktop</launchable>
   <description>
     <p>
       Freedoom: Phase 1 contains four episodes, nine levels each, to
diff --git a/dist/io.github.freedoom.Phase2.metainfo.xml b/dist/io.github.freedoom.Phase2.metainfo.xml
index 02afd455..b5b90e9b 100644
--- a/dist/io.github.freedoom.Phase2.metainfo.xml
+++ b/dist/io.github.freedoom.Phase2.metainfo.xml
@@ -10,6 +10,7 @@
   <developer_name>Freedoom Project</developer_name>
   <url type="homepage">https://freedoom.github.io/</url>
   <url type="bugtracker">https://github.com/freedoom/freedoom/issues</url>
+  <launchable type="desktop-id">io.github.freedoom.Phase2.desktop</launchable>
   <description>
     <p>
       Freedoom: Phase 2 is a 32-level game, expanding upon Phase 1 and
-- 
2.47.1

